vmdb/app/models/dialog_field_drop_down_list.rb

```diff
@@ -1,2 +1,55 @@
 class DialogFieldDropDownList < DialogFieldSortedItem
+  has_one :resource_action, :as => :resource, :dependent => :destroy
+
+  after_initialize :default_resource_action
```

Comment:
@eclarizio Should this logic be in the base class now.  I think only one dialog field is not using it and we plan to convert that one to be dynamic as well.